Applications/Welcome/main.cpp

```diff
@@ -0,0 +1,150 @@
+#include <AK/String.h>
+#include <AK/Vector.h>
+#include <LibDraw/PNGLoader.h>
+#include <LibGUI/GApplication.h>
+#include <LibGUI/GBoxLayout.h>
+#include <LibGUI/GButton.h>
+#include <LibGUI/GDesktop.h>
+#include <LibGUI/GLabel.h>
+#include <LibGUI/GStackWidget.h>
+#include <LibGUI/GWindow.h>
+
+#include "TextWidget.h"
+
+extern const u8 _binary_background_png_start[];
+extern const u8 _binary_background_png_size;
+
+struct ContentPage {
+    String menu_name;
+    String title;
+    Vector<String> content;
```

Comment:
In a perfect world, this would just be a string with `"\n\n"` to separate paragraphs.